Packs/PaloAltoNetworks_IoT3rdParty/Playbooks/Bulk_Export_Devices_to_ServiceNow_-_PANW_IoT_3rd_Party_Integration_README.md

```diff
@@ -0,0 +1,34 @@
+This playbook should be used in the Job to get all available device inventory from PANW IoT Cloud and updates/creates endpoints with custom attributes on ServiceNow.
```

Comment:
Gets the entire available device inventory from PANW IoT Cloud and updates/creates endpoints with custom attributes in ServiceNow. You should run this playbook as a scheduled job.

{I would include an example of a job schedule and why, such as:
For example, you might want to schedule the job to run this playbook at the end of each day so that ServiceNow is updated on a daily basis with the IoT device inventory.}